dwave/cloud/config.py

```diff
@@ -103,30 +103,29 @@
     configuration file::
 
         [defaults]
-        endpoint = https://url.of.some.dwavesystem.com/sapi
-        client = qpu
-
-        [dw2000A]
-        solver = EXAMPLE_2000Q_SYSTEM_A
         token = ABC-123456789123456789123456789
 
+        [primary-qpu]
+        solver = {"qpu": true}
+
         [sw_solver]
```

Comment:
Let's make profile names consistent. Now we have `a-b`, `a_b` and `aB`.

Reply:
Done